packages/fxa-settings/src/components/Avatar/index.stories.tsx

```diff
@@ -3,20 +3,32 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import Avatar from '.';
-import { AppContext } from 'fxa-settings/src/models';
+import { Account, AppContext } from 'fxa-settings/src/models';
+import { Meta } from '@storybook/react';
+
+export default {
+  title: 'Components/Avatar',
+  component: Avatar,
+} as Meta;
 
 const account = {
   avatar: {
     url: null,
     id: null,
   },
 } as any;
-storiesOf('Components/Avatar', module)
-  .add('default avatar', () => (
+
+const storyWithContext = (account: Account, storyName?: string) => {
+  const story = () => (
     <AppContext.Provider value={{ account }}>
       <Avatar className="w-32 h-32" />
     </AppContext.Provider>
-  ))
-  .add('non-default avatar', () => <Avatar className="w-32 h-32" />);
+  );
+  if (storyName) story.storyName = storyName;
+  return story;
+};
```

Comment:
When #14247 is merged, I think it might be nice to remove the `account` object defined here and do something like this:

```
export const DefaultAvatar = storyWithContext(account: MOCK_PROFILE_EMPTY.avatar);
export const NonDefaultAvatar = storyWithContext(account: MOCK_PROFILE_ALL.avatar);
```

Taking it one step further, it'd be even nicer to create a `mocks.tsx` file for `Avatar`, with a couple consts defined like `MOCK_AVATAR_DEFAULT` and `MOCK_AVATAR_NONDEFAULT`, and use those in place of the `MOCK_PROFILE` bits in my example. I could then take your new mocked avatar consts and use them in my mock profile data constants. 

This is a kind of new pattern we're trying to establish with all of our components having a `mocks.tsx` file and I'd love for us to consolidate our mocks in this way instead of having a new `account` object defined in all of our tests/storybooks, so it's not something super obvious.

Reply:
Btw, @millsoper you might already know this but unless something's changed, a reminder we're going to have to [follow these steps](https://mozilla.github.io/ecosystem-platform/reference/team-processes/development-process#merging-a-contributors-pr) to merge Valerie's PRs.